### PR TITLE
footbal match

### PR DESCRIPTION
**Pull request by Vinglish**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -14,7 +14,9 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public static int matchResult(int result1, int result2, int bet1, int bet2) {
+        return (result1 == bet1 && result2 == bet2) ? 2 : (result1 > result2 && bet1 > bet2)
+                ? 1 : (result1 < result2 && bet1 < bet2) ? 1
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: try to use only one "? 1" construction. hint: use || operator

Comment (Vinglish) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -14,7 +14,9 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public static int matchResult(int result1, int result2, int bet1, int bet2) {
+        return (result1 == bet1 && result2 == bet2) ? 2 : (result1 > result2 && bet1 > bet2)
+                ? 1 : (result1 < result2 && bet1 < bet2 || result1 == result2 && bet1 == bet2)
+                ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: rewrite to use "? 1" only once

Comment (Vinglish) [reply src/main/java/core/basesyntax/FootballMatch.java]: My bad. Done.

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**